spec/ttml1.xml

```diff
@@ -4324,26 +4322,57 @@ glyphs do not have an EM square that is distinct from their font's EM square.</p
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+
+<p>Relative <loc href="#style-value-length">&lt;length&gt;</loc> values
```

Comment:
This and the following paragraph should match the presentation order in TTML2, i.e., go before the paragraph proscribing negative length values.

Reply:
Moved to match TTML1 placement